hass_nabucasa/remote.py

```diff
@@ -283,6 +291,8 @@ async def connect(self) -> None:
             _LOGGER.error("Connection problem to snitun server")
         except RemoteBackendError:
             _LOGGER.error("Can't refresh the snitun token")
+        except SubscriptionExpired:
+            pass
```

Comment:
Do we communicate this state some other way to interested parties?

Reply:
No. We just have this property which is updated once an hour when we update auth tokens. Future PR we can add some logic for responding to that.